reference/kubectl-plugin.md

```diff
@@ -2,6 +2,13 @@
 
 The ‘Mayastor kubectl plugin’ can be used to view and manage Mayastor resources such as nodes, pools and volumes. It is also used for operations such as scaling the replica count of volumes. 
 
+### Prerequisites
+
+- Ensure that port <b>30011</b> is open. This port will be needed by Mayastor kubectl plugin to communicate to REST servers from outside the cluster.
+
+{% hint style="warning" %}
+The plugin requires access to the `Mayastor REST server` for execution. It gets the master node IP from the kube-config file. In case of any failure, the REST endpoint can be specified using the ‘–rest’ flag.
```

Comment:
I think that this flag, if used in full, is correctly given as --rest (i.e. two hyphens)
single version is: -r

I also suggest rephrasing this slightly, as this isn't going to resolve just ANY failure.  Perhaps something like "The plugin requires access to the `Mayastor REST server` for execution.  It can usually obtain the correct end point itself, from the kube-config file.  However, if this is not possible, you should specify the master node's IP manually, through the use of the `--rest` or `-r` flag